### PR TITLE
DDP-6897 add ability to edit mailing address from dashboard

### PR DESCRIPTION
**Pull request by aglushko-broad**
image
image

### CODE REVIEW

File: ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ import {
 import { map, take, filter, switchMap, tap, mergeMap, finalize, catchError, mapTo } from 'rxjs/operators';
 import { combineLatest, forkJoin, Observable, of, Subject } from 'rxjs';
 import { TranslateService } from '@ngx-translate/core';
+import { UserPreferencesComponent } from '../../../../../ddp-sdk/src/lib/components/user/userPreferences.component';
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts]:
```suggestion
import { UserPreferencesComponent } from 'ddp-sdk';
```

**Comment by MocanaAtBroad** [comment]
We probably should resize things so if there is a Suggestion from EasyPost, its display is easily visible:
Screen Shot 2021-09-30 at 11 13 43 AM

**Comment by MocanaAtBroad** [comment]
(Let's not forget styling the buttons either.

**Comment by MocanaAtBroad** [comment]
Also, we should play a little bit with indentation/font size/maybe even background color to establish a little more contrast between the dialog title and the mail address component. Maybe Erin can provide some help here.
Screen Shot 2021-09-30 at 11 22 24 AM
.

**Comment by MocanaAtBroad** [comment]
Alignment seems off in mobile (I don't think you have had much of a chance to do styling but flagging it here for now
Screen Shot 2021-09-30 at 11 25 43 AM
)

**Review by MocanaAtBroad — COMMENTED** [review]
Functionality looks good. Need to nail down styling. As I mentioned in Slack, the position of the "Edit" button seems confusing to me/doesn't give any indication of what will be edited. Sizing and styling need to be straightened out too.

File: ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts]: thanks! done

**Comment by aglushko-broad** [comment]
> We probably should resize things so if there is a Suggestion from EasyPost, its display is easily visible: Screen Shot 2021-09-30 at 11 13 43 AM

resize wouldn't help mich here, though I increased the width in 100px. I implemented scrolling to the bottom once suggestion or form error is appeared

**Comment by aglushko-broad** [comment]
> (Let's not forget styling the buttons either.

done!

**Comment by aglushko-broad** [comment]
> Alignment seems off in mobile (I don't think you have had much of a chance to do styling but flagging it here for now Screen Shot 2021-09-30 at 11 25 43 AM )

thanks! fixed

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**